analyses/cms-open-data-ttbar/utils/__init__.py

```diff
@@ -1,12 +1,17 @@
 import asyncio
 import json
+import yaml
 
 import hist
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 import uproot
 from servicex import ServiceXDataset
 import numpy as np
+import awkward as ak
+
+with open("config.yaml") as config_file:
+    config = yaml.safe_load(config_file)
```

Comment:
Could we instead pass in this config information via arguments to the new functions that use it? The main notebook has that config loaded anyway, and I think this will help avoid future issues where we may import functionality from this module from a place that may not have the config available.